tests/newman/files/admin-api-test.csv

```diff
@@ -1,161 +1,16165 @@
-2019,Calendar year,Regional,E92000001,England,,,,E12000002,North West,,,,,,,,,,,,1695
-2015,Calendar year,Regional,E92000001,England,,,,E12000005,West Midlands,,,,,,,,,,,,5872
-2018,Calendar year,Regional,E92000001,England,,,,E12000003,Yorkshire and the Humber,,,,,,,,,,,,4496
-2010,Calendar year,Regional,E92000001,England,,,,E12000001,North East,,,,,,,,,,,,2266
-2019,Calendar year,Regional,E92000001,England,,,,E12000002,North West,,,,,,,,,,,,6296
-2008,Calendar year,Regional,E92000001,England,,,,E12000005,West Midlands,,,,,,,,,,,,5901
-2009,Calendar year,Regional,E92000001,England,,,,E12000003,Yorkshire and the Humber,,,,,,,,,,,,4157
-2015,Calendar year,Regional,E92000001,England,,,,E12000001,North East,,,,,,,,,,,,7731
-2012,Calendar year,Regional,E92000001,England,,,,E12000002,North West,,,,,,,,,,,,6383
-2011,Calendar year,Regional,E92000001,England,,,,E12000005,West Midlands,,,,,,,,,,,,7904
-2017,Calendar year,Regional,E92000001,England,,,,E12000003,Yorkshire and the Humber,,,,,,,,,,,,1615
-2016,Calendar year,Regional,E92000001,England,,,,E12000001,North East,,,,,,,,,,,,9207
-2009,Calendar year,Regional,E92000001,England,,,,E12000002,North West,,,,,,,,,,,,2848
-2009,Calendar year,Regional,E92000001,England,,,,E12000005,West Midlands,,,,,,,,,,,,2192
-2017,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05000364,Syon,1959
-2017,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05006937,Yoxall,9857
-2005,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05010291,Nailsea Youngwood,3612
-2011,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05010450,Rural West York,4415
-2007,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05000364,Syon,9914
-2016,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05006937,Yoxall,6772
-2016,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05010291,Nailsea Youngwood,4198
-2015,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05010450,Rural West York,4613
-2010,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05000364,Syon,6060
-2014,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05006937,Yoxall,3646
-2010,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05010291,Nailsea Youngwood,9304
-2020,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05010450,Rural West York,6650
-2012,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05000364,Syon,1109
-2018,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05006937,Yoxall,1926
-2012,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05010291,Nailsea Youngwood,8150
-2016,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05010450,Rural West York,4035
-2008,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05000364,Syon,5505
-2007,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05006937,Yoxall,7499
-2011,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05010291,Nailsea Youngwood,9603
-2007,Calendar year,Ward,E92000001,England,,,,,,,,,,,,,,,E05010450,Rural West York,9884
+{
```

Comment:
This doesn't look right!

Reply:
haha :)  No it does not - thanks for the assist!!

Reply:
Looks like @markhiveit fixed up already - my thanks!